---
layout: page
description: I am an Associate Professor of Biostatistics at University of Florida.
---

<div class="navbar">
  <div class="navbar-inner">
      <ul class="nav">
	   <li><a href="https://directory.ufhealth.org/chen-li-1">UF website</a></li>
          <li><a href="https://scholar.google.com/citations?user=Nk-yRrcAAAAJ&hl=en">Google Scholar</a></li>
      </ul>
  </div>
</div>



###  Biography
<div class="container">
    <div class="row-fluid">
        <div class="span5">
            Li Chen<br/>
            <a href="https://directory.ufhealth.org/chen-li-1">Associate Professor</a><br/>
		<a href="https://biostat.ufl.edu/">Department of Biostatistics</a><br/>
		 <a href="https://www.ufl.edu/">University of Florida</a><br/>
 2004 Mowry Road, 5th Floor CTRB, 5211 RM  <br/>
    Gainesville, FL 32611-7450 <br/>
            USA<br/>
            <div id="hide_email">
            Email: <code>li.chen1@ufl.edu</code><br/>
            </div>
        </div>
</div>
<br/>
	</div>


I am an Associate Professor of Biostatistics at University of Florida. I obtained my PhD in [Computer Science and Informatics](http://www.cs.emory.edu/) from Emory University, and a MS in [Biostatistics](http://www.jhsph.edu/departments/biostatistics/index.html) and another MS in [Computer Science](https://www.cs.jhu.edu/) from the Johns Hopkins University.

### Opening
My lab now has multiple openings for PhD/MS students and one opening for Postdoc. Interested candidates please refer the [Opening](https://lichen-lab.github.io//pages/opening.html) for how to apply.

### Research
My lab is currently funded by NIH NIGMS. My lab's research focuses on developing statistical and informatics methods for analyzing of multi-omics data. Projects include but not limited to 

- **Epigenomics**: 
Develop statistical methods and database for analyzing epigenomic data
	- Differential binding. [<a href="https://academic.oup.com/bioinformatics/article/31/12/1889/214643">ChIPComp</a>]
	- TF regulatory module. [<a href="https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/btz975/5695705">tfLDA</a>]
	- Database and web server. [<a href="https://academic.oup.com/bioinformatics/article/27/10/1447/260530">hmChIP</a>]
	- Predictive modeling of disease status. [<a href="https://www.sciencedirect.com/science/article/pii/S2001037022004639?via%3Dihub">MTAE</a>]
	- Predicting 3D chromatin interaction. [<a href="https://www.biorxiv.org/content/10.1101/2022.05.24.493333v1">DeepPHiC</a>]

- **Genetics**:
Develop computational methods for annotating and analyzing functional variants (GWAS SNPs, cis-eQTL SNVs) by utilizing large-scale multi-omics profiles
	- GWAS variant enrichment analysis. [<a href="https://academic.oup.com/bioinformatics/article/32/8/1214/1744655">traseR</a>]
	- Functional variant prediction. [<a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-016-1112-z">DIVAN</a>;<a href="https://academic.oup.com/bioinformatics/article/35/9/1573/5126236">TIVAN</a>;
	 <a href="https://academic.oup.com/bib/advance-article-abstract/doi/10.1093/bib/bbab189/6279833?redirectedFrom=fulltext">WEVar</a>;
	 <a href="https://pubmed.ncbi.nlm.nih.gov/35389435/">TLVar</a>;
	 <a href="https://pubmed.ncbi.nlm.nih.gov/36271868/">DeepPerVar</a>]

- **Metagenomics**:
Develop statistical methods for analyzing microbiome data
	- Normalization. [<a href="https://peerj.com/articles/4600/">GMPR</a>]
	- Power analysis. [<a href="https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/btaa197/5809526">powmic</a>]
	- Predictive modeling.[<a href="https://academic.oup.com/bioinformatics/article/31/24/3991/197681">glmgraph</a>; <a href="https://www.frontiersin.org/articles/10.3389/fmicb.2018.03112/full">SICS</a>; <a href="https://www.frontiersin.org/articles/10.3389/fmicb.2018.01391/full">glmmTree</a>;
	<a href="https://academic.oup.com/bib/advance-article-abstract/doi/10.1093/bib/bbaa073/5835556?redirectedFrom=fulltext">MDeep</a>]
	
- **Single-cell genomics**:
Develop statistical methods for clustering single-cell data
	- Clustering scRNA data.[<a href="https://www.nature.com/articles/s41467-019-09639-3">BAMMSC</a>]

- **Transcriptomics**:
Develop statistical methods for analyzing non-coding RNAs
	- Differential expression analysis. [<a href="https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/btz606/5543088">circMeta</a>]


