---
layout: page
description: I am a tenure-track Assistant Professor of Medicine at Indiana University School of Medicine. I am also a member at Center for Computational Biology and Bioinformatics.
---

<div class="navbar">
  <div class="navbar-inner">
      <ul class="nav">
          <li><a href="https://scholar.google.com/citations?user=Nk-yRrcAAAAJ&hl=en">Google Scholar</a></li>
        <li><a href="">Official Website</a></li>
      </ul>
  </div>
</div>



<div class="container">
    <div class="row-fluid">
            Li Chen, Principal Investigator<br/>
            <a href="https://medicine.iu.edu/departments/internal-medicine/specialties/hematology-oncology/faculty/47520/chen-li/">Assistant Professor of Medicine</a><br/>
	    Member, <a href="https://medicine.iu.edu/research/centers-institutes/computational-biology-bioinformatics/">Center for Computational Biology and Bioinformatics, Indiana University School of Medicine</a><br/>
    </div>
</div>



###  Biography

I am a tenure-track Assistant Professor of Medicine at Indiana University School of Medicine (IUSM). I am a member in Center for Computational Biology and Bioinformatics. Before joining IUSM, I spent two years as an Assistant Professor in Harrison School of Pharmacy at Auburn University. I received a BS from China majoring in Bioinformatics in 2009; a MHS in [Biostatistics](http://www.jhsph.edu/departments/biostatistics/index.html) in 2011, from the Johns Hopkins University; another MSE in [Computer Science](https://www.cs.jhu.edu/) in 2011, from the Johns Hopkins University. After that, I obtained my PhD in [Computer Science and Informatics](http://www.cs.emory.edu/site/home/) at Emory University in 2017. I also interned in CareerBuilder Data Science and Amazon Machine Learning in Summer 2014 and 2015 respectively.


### Research
I am interested in data science research with its application in biomedical data especially large-scale omics data and population-based epidemiological data. I am leading a small interdisciplinary data science group working on bioinformatics & biostatistics projects. My current research focuses on developing statistical and informatics methods for analyzing of large-scale omics and population-based epidemiological data. Projects include but not limited to 

- **Epigenomics**: 
Develop statistical methods and database for epigenomic data in differential biological conditions.
	- [Differential binding](https://academic.oup.com/bioinformatics/article-lookup/doi/10.1093/bioinformatics/btv094). [<a href="http://bioconductor.org/packages/devel/bioc/html/ChIPComp.html">Link to ChIPComp</a>]
	- [Database and web server](https://academic.oup.com/bioinformatics/article/27/10/1447/260530). [<a href="http://jilab.biostat.jhsph.edu/database/cgi-bin/hmChIP.pl">Link to hmChIP</a>]

- **Genetics**:
Develop computational methods for annotating and predicting functional variants (GWAS SNPs, cis-eQTL SNVs) by utilizing large-scale multi-omics profiles. 
	- [GWAS variant enrichment analysis](https://academic.oup.com/bioinformatics/article/32/8/1214/1744655). [<a href="http://bioconductor.org/packages/release/bioc/html/traseR.html">Link to traseR</a>]
	- [GWAS variant prediction](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-016-1112-z). [<a href="https://sites.google.com/site/emorydivan/">Link to DIVAN</a>]
	- [cis-eQTL variant prediction](https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/bty872/5126236). [<a href="https://github.com/lichen-lab/TIVAN">Link to TIVAN</a>]

- **Metagenomics**:
Develop statistical methods for predictive modeling disease status based on 16S rRNA sequencing microbiome data.
	- [Normalization](https://peerj.com/articles/4600/). [<a href="https://github.com/lichen-lab/GMPR">Link to GMPR</a>]
	- [Predictive modeling using Graph-Constraint Sparse Regression Model](https://academic.oup.com/bioinformatics/article/31/24/3991/197681). [<a href="https://cran.r-project.org/web/packages/glmgraph/index.html">Link to glmgraph</a>]
	- [Predictive modeling using Graph-Constraint Sparse Regression Model with encouraging local smoothing](https://www.frontiersin.org/articles/10.3389/fmicb.2018.03112/abstract). [<a href="https://github.com/lichen-lab/SICS">Link to SICS</a>]
	- [Predictive modeling using Generalized Linear Mixed Model](https://www.frontiersin.org/articles/10.3389/fmicb.2018.01391/abstract). [<a href="https://github.com/lichen-lab/glmmTree">Link to glmmTree</a>]
	
- **Single-cell genomics**:
Develop statistical methods for clustering single cell  data.
	- [Clustering droplet-based single cell transcriptomic data from population studies](https://www.nature.com/articles/s41467-019-09639-3). [<a href="https://github.com/lichen-lab/BAMMSC">Link to BAMMSC</a>]

- **Transcriptomics**:
Develop statistical methods for non-coding RNA.
	- [Genomic feature annotation and differential expression analysis of circular RNAs](https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/btz606/5543088). [<a href="https://github.com/lichen-lab/circMeta">Link to circMeta</a>]


### Contact
<div class="container">
    <div class="row-fluid">
        <div class="span5">
            Li Chen<br/>
            <a href="https://medicine.iu.edu/departments/internal-medicine/specialties/hematology-oncology/faculty/47520/chen-li/">Department of Medicine, Indiana University School of Medicine</a><br/>
            <a href="https://medicine.iu.edu/research/centers-institutes/computational-biology-bioinformatics/">Center for Computational Biology and Bioinformatics, Indiana University School of Medicine</a><br/>
 410 W. 10th Street, HS 5025 <br/>
    Indianapolis, IN 46202<br/>
            USA<br/><br/>
            <div id="hide_email">
            Email: <code>chen61@iu.edu</code><br/>
            </div>
        </div>

        <div class="span2">
        <a href="../assets/pics/Chen.jpg">
            <img src="../assets/pics/Chen.jpg" height="200" width="150" title="Li Chen" alt="Li Chen"/>
        </a>
        </div>
    </div>
</div>


