---
layout: page
title: Software
description: Li Chen's software
---

Here is a list of software I developed. 

---

### Deep learning methods for analyzing multiome data:
* <a name="DeepPHiC"></a>[DeepPHiC](https://github.com/lichen-lab/DeepPHiC)
:  A Python package for implementing a novel multi-task and transfer deep learning model to predicting promoter-centered chromatin interactions using promoter-centered Hi-C data.
* <a name="MTAE"></a>[MTAE](https://github.com/lichen-lab/MTAE)
: A Python package for implementing a novel multi-task deep autoencoder to predict AD progression using longitudinal DNA methylation data in peripheral blood.
* <a name="TLVar"></a>[TLVar](https://github.com/lichen-lab/TLVar)
: A Python package for implementing a deep transfer learning approach to improve the prediction for experimentally validated regulatory variants.
* <a name="DeepPerVar"></a>[DeepPerVar](https://github.com/lichen-lab/DeepPerVar)
: A Python package for implementing a multi-modal deep learning model to predict individual-level noncoding functional variants.
* <a name="MDeep"></a>[MDeep](https://github.com/lichen-lab/MDeep)
: A Python package for implementing a novel deep learning model to predict phenotype using microbiome data by embedding the phylogenetic tree in the deep learning model.

---

### Bayesian methods for analyzing multiome data:
* <a name="BayesDCirc"></a>[BAMMSC](https://github.com/lichen-lab/BayesDCirc)
: An R package for implementing a Bayesian hierarchical model for detecting differential circadian pattern in transcriptomic applications
* <a name="BAMMSC"></a>[BAMMSC](https://github.com/lichen-lab/BAMMSC)
: An Rpackage for implementing a novel Bayesian mixture model to cluster droplet-based single cell transcriptomic
data from multiple individuals.
* <a name="scaDA"></a>[scaDA](https://github.com/lichen-lab/scaDA)
: An R package for implementing an empirical bayesian methods for differential accessibility analysis using single cell multiome data

---

### Spatial multiome data analysis:
* <a name="Sami"></a>[Sami](https://github.com/XinBiostats/SAMI)
: A python toolkit for simultaneous spatial analysis of the metabolome, lipidome, and glycome from a single tissue section using mass spectrometry imaging.
* <a name="MetaVision3D"></a>[MetaVision3D](https://github.com/XinBiostats/MetaVision3D)
: A Python toolkit uses advanced algorithms for image registration, normalization and interpolation to enable the integration of serial 2D tissue sections, thereby generating a comprehensive 3D model of unique diverse metabolites across host tissues at submesoscale.

---

### Single-cell multiome data analysis:
* <a name="BAMMSC"></a>[BAMMSC](https://github.com/lichen-lab/BAMMSC)
: An Rpackage for implementing a novel Bayesian mixture model to cluster droplet-based single cell transcriptomic
data from multiple individuals.
* <a name="scaDA"></a>[scaDA](https://github.com/lichen-lab/scaDA)
: An R package for implementing an empirical bayesian methods for differential accessibility analysis using single cell multiome data

---

### Functional noncoding variants prioritization:
* <a name="traseR"></a>[traseR](http://bioconductor.org/packages/release/bioc/html/traseR.html)
: An R package for performing GWAS trait-associated SNP enrichment analyses in given genomic intervals.
* <a name="DIVAN"></a>[DIVAN](https://sites.google.com/site/emorydivan/)
: An R pipeline for prioritizing disease-specific noncoding risk variants in 45 diseases/traits using genome-wide precomputed functional scores.
* <a name="WEVar"></a>[WEVar](https://github.com/lichen-lab/WEVar)
: A Python package for improving the prediction for noncoding regulatory variants using a weighted ensemble approach by integrating precomputed functional scores from multiple existing methods.
* <a name="TIVAN"></a>[TIVAN](https://github.com/lichen-lab/TIVAN)
: An R pipeline for predicting noncoding regulatory variants in 44 tissues/cell types.
* <a name="TIVAN-indel"></a>[TIVAN-indel](https://github.com/lichen-lab/TIVAN-indel)
: A Python package for implementing a novel computational method for predicting noncoding regulatory small insertions and deletions.
* <a name="TLVar"></a>[TLVar](https://github.com/lichen-lab/TLVar)
: A Python package for implementing a deep transfer learning approach to improve the prediction for experimentally validated regulatory variants.
* <a name="DeepPerVar"></a>[DeepPerVar](https://github.com/lichen-lab/DeepPerVar)
: A Python package for implementing a multi-modal deep learning model to predict individual-level noncoding functional variants.

---

### Microbiome data analysis:
* <a name="GMPR"></a>[GMPR](https://github.com/lichen-lab/GMPR)
: An R package for normalizing zero-inflated count data particular microbiome sequencing data.
* <a name="glmgraph"></a>[glmgraph](http://cran.r-project.org/web/packages/glmgraph/index.html)
: An R package for implementing sparse generalized linear models with graph-constrained regularization.
* <a name="SICS"></a>[SICS](https://github.com/lichen-lab/SICS)
: An R package for implementing sparse generalized linear models with encouraging local smoothing in a phylogeny-constrained regularization for predictive modeling of microbiome data.
* <a name="glmmTree"></a>[glmmTree](https://github.com/lichen-lab/glmmTree)
: An R package for implementing a phylogenetic tree-based generalized mixed effects model for predictive modeling of microbiome data.
* <a name="powmic"></a>[powmic](https://github.com/lichen-lab/powmic)
: An R package for performing power assessment in microbiome sequencing data.
* <a name="MDeep"></a>[MDeep](https://github.com/lichen-lab/MDeep)
: A Python package for implementing a novel deep learning model to predict phenotype using microbiome data by embedding the phylogenetic tree in the deep learning model.

---


### Epigenetic data analysis:
* <a name="DeepPHiC"></a>[DeepPHiC](https://github.com/lichen-lab/DeepPHiC)
:  A Python package for implementing a novel multi-task and transfer deep learning model to predicting promoter-centered chromatin interactions using promoter-centered Hi-C data.
* <a name="MTAE"></a>[MTAE](https://github.com/lichen-lab/MTAE)
: A Python package for implementing a novel multi-task deep autoencoder to predict AD progression using longitudinal DNA methylation data in peripheral blood.
* <a name="tfLDA"></a>[tfLDA](https://github.com/lichen-lab/tfLDA)
: An R package for applying state-of-the-art topic models to decipher the combinatorial binding events of multiple TFs by integrating multiple ChIP- Seq datasets.
* <a name="ChIPComp"></a>[ChIPComp](https://bioconductor.org/packages/release/bioc/html/ChIPComp.html)
: An R package for implementing a novel Bayesian hierarchical model for quantitative comparison of multiple ChIP-seq datasets.
* <a name="hmChIP"></a>[hmChIP](http://jilab.biostat.jhsph.edu/database/cgi-bin/hmChIP.pl)
: A web server and database for cell type-specific ChIP-seq and ChIP-chip data query.
---

### Noncoding RNA analysis:
* <a name="circMeta"></a>[circMeta](https://github.com/lichen-lab/circMeta)
:  An R package for detecting differential expression analysis of circular RNAs for small biological replicates.
* <a name="circMeta2"></a>[circMeta2](https://github.com/lichen-lab/circMeta2)
:  An R package for detecting differential expression analysis of circular RNAs for large-scale population study.

---










