---
layout: page
title: Software
description: Li Chen's software
---


Here is a list of software I developed or co-developed. 


### R CRAN/Bioconductor packages

* <a name="ChIPComp"></a>[ChIPComp](http://bioconductor.org/packages/release/bioc/html/ChIPComp.html)
:differential protein binding/histone modification analysis for ChIP-seq data.

* <a name="traseR"></a>[traseR](http://bioconductor.org/packages/release/bioc/html/traseR.html)(**TR**ait-**A**ssociated **S**NP **E**n**R**ichment analysis)
:disease/traits enrichment analyses of trait-associated SNPs in arbitrary genomic intervals.

* <a name="glmgraph"></a>[glmgraph](https://cran.r-project.org/web/packages/glmgraph/index.html)(**GRAPH**-constrained regularization for sparse **G**eneralized **L**inear **M**odels)
:network/tree-based penalized regression method. This method could be useful in high-dimensional, sparse data prediction when the gragh/network structure among predictors are known.

---


### github software packages

* <a name="glmmTree"></a>[glmmTree](https://github.com/lichen-lab/glmmTree)
:an R package for predictive modeling of microbiome data using a phylogenetic tree-based generalized mixed effects model

* <a name="SICS"></a>[SICS](https://github.com/lichen-lab/SICS)
:an R package for predictive modeling of microbiome data using a phylogeny-constrained regularization for sparse generalized linear models with encouraging local smoothing

* <a name="GMPR"></a>[GMPR](https://github.com/lichen-lab/GMPR)
:an R package for normalizing zero-inflated count data particular microbiome sequencing data

* <a name="BAMMSC"></a>[BAMMSC](https://github.com/lichen-lab/BAMMSC)
:an R package for a bayesian mixture model for clustering droplet-based single cell transcriptomic data from population studies

* <a name="circMeta"></a>[circMeta](https://github.com/lichen-lab/circMeta)
:an R package for genomic feature annotation and differential expression analysis of circular RNAs

---

### Database

* <a name="hmChIP"></a>[hmChIP](http://jilab.biostat.jhsph.edu/database/cgi-bin/hmChIP.pl)
:a database of genome-wide ChIP-seq data in human and mouse. 

*  <a name="DIVAN"></a>[DIVAN](https://sites.google.com/site/emorydivan/)
:a database/pipeline for disease-specific noncoding variant annotation and prioritization.

*  <a name="TIVAN"></a>[TIVAN](https://github.com/lichen-lab/TIVAN)
:a database/pipeline for tissue-specific cis-eQTL variants annotation and prioritization.









