---
layout: page
title: Software
description: Li Chen's software
---


Here is a list of software I developed. 


### R CRAN/Bioconductor packages

* <a name="ChIPComp"></a>[ChIPComp](http://bioconductor.org/packages/release/bioc/html/ChIPComp.html)
: differential protein binding/histone modification analysis for ChIP-seq data

* <a name="traseR"></a>[traseR](http://bioconductor.org/packages/release/bioc/html/traseR.html)
: disease/traits enrichment analyses of trait-associated SNPs

* <a name="glmgraph"></a>[glmgraph](https://cran.r-project.org/web/packages/glmgraph/index.html)
: network/tree-based penalized regression method

---


### github R/Python software packages

* <a name="glmmTree"></a>[glmmTree](https://github.com/lichen-lab/glmmTree)
: predictive modeling of microbiome data using a phylogenetic tree-based GLMM

* <a name="SICS"></a>[SICS](https://github.com/lichen-lab/SICS)
: predictive modeling of microbiome data using a phylogeny-constrained regularization for sparse GLM

* <a name="GMPR"></a>[GMPR](https://github.com/lichen-lab/GMPR)
: normalizing zero-inflated count data particularly microbiome sequencing data

* <a name="BAMMSC"></a>[BAMMSC](https://github.com/lichen-lab/BAMMSC)
: bayesian mixture model for clustering droplet-based scRNA-seq data from population studies

* <a name="circMeta"></a>[circMeta](https://github.com/lichen-lab/circMeta)
: genomic feature annotation and differential expression analysis of circular RNAs

* <a name="tfLDA"></a>[tfLDA](https://github.com/lichen-lab/tfLDA)
: topic models to multiple ChIP-Seq datasets to decipher the combinatorial binding of TFs

* <a name="powmic"></a>[powmic](https://github.com/lichen-lab/powmic)
: power assessment in microbiome case-control studies

* <a name="MDeep"></a>[MDeep](https://github.com/lichen-lab/MDeep)
: predictive modeling of microbiome data using a novel convolutional neural network

* <a name="TLVar"></a>[TLVar](https://github.com/lichen-lab/TLVar)
: Exploiting deep transfer learning for the prediction of functional noncoding variants using genomic sequence

* <a name="TLVar"></a>[TLVar-indel](https://github.com/lichen-lab/TLVar-indel)
: Exploiting deep transfer learning for the prediction of functional noncoding small insertation/deletion using genomic sequence

* <a name="MTAE"></a>[MTAE](https://github.com/lichen-lab/MTAE)
: Multi-task deep autoencoder improves the prediction for Alzheimer's disease progression using temporal DNA methylation data in peripheral blood

* <a name="DeepPerVar"></a>[DeepPerVar](https://github.com/lichen-lab/DeepPerVar)
: By leveraging paired whole genome sequencing data and epigenetic functional assays in a population cohort, a DeepPerVar is a multi-modal deep learning framework to predict genome-wide quantitative epigenetic signals and evaluate the functional consequence of noncoding variants on an individual level by quantifying their allelic difference on the prediction. 

* <a name="DeepPHiC"></a>[DeepPHiC](https://github.com/lichen-lab/DeepPHiC)
: DeepPHiC: Predicting promoter-centered chromatin interactions using a deep learning approach

---

### Database

* <a name="hmChIP"></a>[hmChIP](http://jilab.biostat.jhsph.edu/database/cgi-bin/hmChIP.pl)
: genome-wide ChIP-seq data in human and mouse 

*  <a name="DIVAN"></a>[DIVAN](https://sites.google.com/site/emorydivan/)
: disease-specific noncoding variant annotation and prioritization

*  <a name="TIVAN"></a>[TIVAN](https://github.com/lichen-lab/TIVAN)
: tissue-specific cis-eQTL variants annotation and prioritization









